coordinator/persistence-dse-6.8/pom.xml

```diff
@@ -135,7 +135,7 @@
     <dependency>
       <groupId>io.netty</groupId>
       <artifactId>netty-codec</artifactId>
-      <version>4.1.78.1.dse</version>
+      <version>4.1.86.1.dse</version>
```

Comment:
Updated as per DSE 6.8.34 release notes (may also resolve Netty CVEs, not 100% sure)